webpack.dev.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { merge } = require('webpack-merge');
+const { shared } = require('./webpack.config');
+
+const config = merge(shared, {
+  entry: {
+    demo: './example/index.jsx',
+  },
+  output: {
+    path: path.resolve(__dirname, 'example/'),
+    filename: '[name].js',
+  },
+  devServer: {
+    contentBase: './example',
+    compress: true,
+    port: 9966,
+    hot: true,
+    watchContentBase: true,
+  },
+  resolve: {
+    alias: {
+      'react-dom': '@hot-loader/react-dom',
+    },
+  },
+  cache: {
+    type: 'filesystem',
+  },
```

Comment:
What's this for?

Reply:
Whoops. I was trying to figure out why webpack was so slow. I had added sourcemaps.

Reply:
Incidentally, loading the markdown files with sourcemaps enabled causes this build to go from around <500ms to 10-15 seconds. Very annyoing.